playwright-e2e/config/.env.dev

```diff
@@ -0,0 +1,17 @@
+# Ensure environment variable names are unique across different apps
```

Comment:
Group all `.env` files in `config` dir. Setting up config for diff. env for CI testing.